code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -587,11 +587,27 @@
 	icon_state = "portafreezer"
 	item_state = "medicalpack"
 	max_w_class = 3
-	can_hold = list(/obj/item/organ, /obj/item/reagent_containers/food, /obj/item/reagent_containers/glass)
 	max_storage_space = 21
-	use_to_pickup = 1 // for picking up broken bulbs, not that most people will try
+	use_to_pickup = 0 // for picking up broken bulbs, not that most people will try
```

Comment:
```suggestion
	use_to_pickup = FALSE // for picking up broken bulbs, not that most people will try
```
Usually this is preferred for vars that act like bools because it makes it clear it can only be in two states.

Reply:
I will say Im a little ashamed because I had the boolean conversation not too long after the initial post. Whoops!